spec/scenic/schema_dumper_spec.rb

```diff
@@ -36,4 +36,48 @@ class Search < ActiveRecord::Base; end
       Search.connection.drop_view :'scenic.searches'
     end
   end
+
+  context "with dependent views " do
+    it "dumps create_view sorted alphabetically" do
+      Search.connection.execute("CREATE SCHEMA scenic")
+      Search.connection.execute("SET search_path TO scenic, public")
+
+      views = {
+        "scenic.view1": "SELECT 'needle1'::text AS haystack",
+        "scenic.view2": "SELECT 'needle2'::text AS haystack",
+        "scenic.view1_1": "SELECT * FROM scenic.view1",
+        "scenic.view1_2": "SELECT * FROM scenic.view1",
+      }
+
+      expected_order = %w(
+        scenic.view1
+        scenic.view1_1
+        scenic.view1_2
+        scenic.view2
+      )
```

Comment:
This is the same order you'd expect from an alphabetical sort. You cover this a bit in postgres_spec but...

Can you complicate things by:

1. Adding something that relies on scenic.view1 and scenic.view1_1
2. Changing some names so that the expected ordering is non-alphabetical